src/svg-pan-zoom.js

```diff
@@ -30,6 +30,10 @@ var optionsDefaults = {
 , customEventsHandler: null
 , eventsListenerElement: null
 , onUpdatedCTM: null
+, triggerUpOnLeave: true // This allows content to be panned even if the panning pointer is on top of a menu or outside browser
+, restrictPanButton: null // This restricts pan to specific evt.button
+, restrictPanWhilePointerIsDown: false // This triggers panning only if the pointer is down (ex: only when mouse button is down)
```

Comment:
I don't really understand why this is necessary. It's weird to want to pan without clicking and holding as in this way the pan will always happen, even if the user doesn't want to interact with the SVG.